libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -455,7 +456,27 @@ bool AP_Arming::hardware_safety_check(bool report)
 
 bool AP_Arming::rc_calibration_checks(bool report)
```

Comment:
Should be `const`; pre-existing issue :-)

Reply:
I let this slip through, on account of needing to then touch copter + sub...